iset.mm

```diff
@@ -152516,14 +152557,45 @@ which membership in it is a bounded proposition ( ~ df-bdc ).
       ZADFKZBCLTAUABCHMBCDSFSNOPACESGQR $.
   $}
 
+  $( With a formula ` ph ` one can associate an element of ` ~P 1o ` , which
+     can therefore be thought of as the set of "truth values" (but recall that
+     there are no other genuine truth values than ` T. ` and ` F. ` , by
+     ~ nndc , which translate to ` 1o ` and ` (/) ` respectively by ~ iftrue
+     and ~ iffalse , giving ~ pwtrufal ).  (Contributed by BJ, 15-Aug-2024.) $)
+  fmelpw1o $p |- if ( ph , 1o , (/) ) e. ~P 1o $=
```

Comment:
We'd also be able to prove `if ( ph , 1o , (/) ) = { x e. 1o | ph }`, right?

(Not a blocking comment, and no need to add it to this pull request unless you want, just an observation).

Reply:
We may also add it indeed. With `fmelpw1o`, I wanted to expose $\mathcal{P}1$ as a particularly important set.

Reply:
Done. I added the general instance of that theorem.